src/Auth0Client.ts

```diff
@@ -773,17 +808,27 @@ export default class Auth0Client {
         60 // get a new token if within 60 seconds of expiring
       );
 
-      return cache && cache.access_token;
+      if (entry && entry.access_token) {
```

Comment:
Wont this skip over this when we have content in the cache but without an access token? Is this expected ?

Reply:
Yes, to be consistent with existing behaviour.

Reply:
Yeah that makes sense, thanks!